alacritty/src/config/mod.rs

```diff
@@ -168,12 +168,24 @@ fn parse_config(contents: &str) -> Result<Config> {
             }
         },
         Ok(config) => {
+            check_working_directory_absolute(&config);
             print_deprecation_warnings(&config);
             Ok(config)
         },
     }
 }
 
+fn check_working_directory_absolute(config: &Config) {
+    if let Some(dir) = &config.working_directory {
+        if !dir.is_absolute() {
+            warn!(
+                target: LOG_TARGET_CONFIG,
+                "Config working_directory is not absolute: {:?}", dir
+            );
+        }
+    }
+}
```

Comment:
This is not related to this PR and thus should not be included here.

Additionally, I think it's not desirable, so I'm not sure if opening a separate PR would prove fruitful.

Reply:
👍 I'll bin it.

Reply:
Just out of curiosity, are you aware of the recent change about printing errors on startup with invalid working directories?

I can see the motivation behind this on top of our latest release, since that has no indication of incorrect working directories and will just crash hard. However with the latest master I feel like we already provide sufficient information.

Reply:
> Oh nice; I was working off a slightly older master originally which didn't have that check! +1

Nevermind, you answered me already. That makes a lot more sense then!